platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationFileDirectory.java

```diff
@@ -119,6 +126,23 @@ public void notify(Path file) {
         }
     }
 
+    public Collection<ConfigurationStatus> getFailedConfigurationFiles() {
+        Collection<ConfigurationStatus> configStatusMessages = new ArrayList<>();
+        try (DirectoryStream<Path> stream = getFailedDirectoryStream()) {
+            for (Path path : stream) {
+                ConfigurationStatus configStatus = new ConfigurationStatus();
+                configStatus.setPath(path.getFileName());
+                configStatusMessages.add(configStatus);
+                LOGGER.debug("Adding [{}] to the failed imports list.", path.toString());
+            }
+        } catch (Exception e) {
```

Comment:
Should we catch `IOException | RuntimeException` here instead of `Exception`?

Also, it looks like the only time we set a message in a `ConfigurationStatus` is when an exception gets thrown in the loop. Do you think we should just throw the exception out of the service if something like that happens since we couldn't complete the operation and the list of `ConfigurationStatus` would be incomplete? And maybe we could get rid of the message altogether, which would simplify the `ConfigurationStatus` class and the command's `constructErrorMessage` method as well.